src/cli.ts

```diff
@@ -194,6 +199,32 @@ program
     'the target Kibana spaces for the pushed monitors — spaces help you organise pushed monitors.'
   )
   .option('-y, --yes', 'skip all questions and run non-interactively')
+  .option(
+    '--proxy-uri <uri>',
+    'proxy uri to use when pushing to kibana',
+    collectOpts('uri', proxySettings)
+  )
+  .option(
+    '--proxy-token <token>',
+    'auth token to use the proxy',
+    collectOpts('token', proxySettings)
+  )
+  .option(
+    '--proxy-ca <path>',
+    'provide a CA override for proxy endpoint, as a path to be loaded or as string',
+    collectOpts('ca', proxySettings, parseFileOption)
+  )
+  .option(
+    '--proxy-cert <path>',
+    'provide a cert override for proxy endpoint, as a path to be loaded or as string',
+    collectOpts('cert', proxySettings, parseFileOption)
+  )
+  .option(
+    '--proxy-no-verify',
+    'disable TLS verification for the proxy connection',
```

Comment:
does overriding NODE_TLS_UNAUTHORIZED disable this? can you callout this in the description.